src/server/src/test/resources/logback-test.xml

```diff
@@ -21,8 +21,18 @@
       <pattern>%d %-5p [%c{0}] %m%n</pattern>
     </encoder>
   </appender>
+  
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <layout class="ch.qos.logback.classic.PatternLayout">
+        <Pattern>
+            %d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} - %msg%n
+        </Pattern>
+    </layout>
+</appender>
+  
   <root>
     <level value="DEBUG" />
     <appender-ref ref="file"/>
+    <!-- <appender-ref ref="STDOUT"/> -->
```

Comment:
i don't think any of this is part of the PR ?

Reply:
nope